README.md

```diff
@@ -70,6 +70,38 @@ services:
 
 You can delete the "dev" core from `http://<projectname>.ddev.site:8983/solr/#/~cores/dev` by clicking "Unload".
 
+## Multiple Solr Cores
+
+If you would like to use more than one Solr core, you'll need to edit the file [`.ddev/docker-compose.solr.yaml`](docker-compose.solr.yaml)
```

Comment:
Rather than editing, which orphans the add-on, use a `docker-compose.solr_extra.yaml`

Reply:
Thanks for the feedback - i've updated the instructions accordingly